ui/src/variables/actions/index.ts

```diff
@@ -141,11 +145,16 @@ export const selectValue = (
   payload: {contextID, variableID, selectedValue},
 })
 
-export const getVariables = () => async (dispatch: Dispatch<Action>) => {
+export const getVariables = () => async (
+  dispatch: Dispatch<Action>,
+  getState: GetState
+) => {
   try {
     dispatch(setVariables(RemoteDataState.Loading))
-
-    const variables = await client.variables.getAll()
+    const {
+      orgs: {org},
+    } = getState()
+    const variables = (await client.variables.getAll(org.id)) as IVariable[]
```

Comment:
why do we need type coercion here? does get all not return it as the type we need?

Reply:
this needs to be fixed in client